src/emulator/storage/index.ts

```diff
@@ -5,24 +5,26 @@ import { EmulatorInfo, EmulatorInstance, Emulators } from "../types";
 import { createApp } from "./server";
 import { StorageLayer } from "./files";
 import { EmulatorLogger } from "../emulatorLogger";
-import { StorageRulesManager } from "./rules/manager";
+import { createStorageRulesManager, StorageRulesManager } from "./rules/manager";
 import { StorageRulesetInstance, StorageRulesRuntime, StorageRulesIssues } from "./rules/runtime";
 import { SourceFile } from "./rules/types";
 import express = require("express");
 import { getAdminCredentialValidator, getRulesValidator } from "./rules/utils";
 import { Persistence } from "./persistence";
 import { UploadService } from "./upload";
 
+export type RulesType = SourceFile | string;
```

Comment:
Can we remove this type and just force everything to be a `SourceFile`? There's already a straightforward conversion from `string` to `SourceFile`, why not convert to `SourceFile` any time we get a new `string` rules?